rho/clicommands.py

```diff
@@ -266,7 +266,8 @@ def _create_ping_inventory(profile_ranges, profile_auth_list, forks):
 
         out = out.split('\n')
 
-        for line in enumerate(out):
+        # pylint: disable=consider-using-enumerate
```

Comment:
I see. I think you want `for line, _ in enumerate(out):`. `enumerate` returns a pair and not an integer, so using its output as an index into a list should have produced an error.